CHEATSHEET.md

```diff
@@ -112,31 +119,33 @@ Restrict sym_arg to typical char range
 ```python
 for byte in sym_arg.chop(8):
     initial_state.add_constraints(byte != '\x00') # null
```

Comment:
This is technically out of the scope of your changes, but this constraint is redundant...